makefiles/pseudomodules.inc.mk

```diff
@@ -1,4 +1,6 @@
 PSEUDOMODULES += at_urc
+PSEUDOMODULES += at24c128
+PSEUDOMODULES += at24c256%
```

Comment:
Why is the `%` needed ?

Reply:
This driver also doesn't only support those two EEPROMs, but any at24xxx device - also from different vendors.

Reply:
> Why is the % needed ?

The `%` was intended for the `at24c265c`.

> This driver also doesn't only support those two EEPROMs, but any AT24Cxxx device - also from different vendors.

It supports those EEPROMs with `<= 65,536 bytes` and `64 bytes page size`.
E.g. not `at24c1024`.

Should I anyway just use  `at24c%`?

Reply:
> Should I anyway just use at24c%?

Yes

Reply:
That would also match `at24cxxx`. Any idea how to handle that?

Reply:
I don't think this is a problem but I'm not sure.